spec/confirmation_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'confirmation'
+
+describe Confirmation do
+  subject(:confirmation)	{ described_class.new }
+	let(:send_text)	        { double(:send_text) }
+
+  describe 'confirmation message' do
```

Comment:
The send_text method has been stubbed in this test, so the test isn't testing anything! Instead, think about what calling send_text should do, and focus on testing that behaviour.